pkg/auth/providers/oidc/oidc_provider.go

```diff
@@ -423,9 +433,6 @@ func (o *OpenIDCProvider) getUserInfo(ctx *context.Context, config *v32.OIDCConf
 	if err != nil {
 		return userInfo, oauth2Token, err
 	}
-	if err := userInfo.Claims(&claimInfo); err != nil {
```

Comment:
Why did you remove this check? I see above you use `idToken.Claims`, but if I understand correctly, `userInfo` and `idToken` are different.

Reply:
Good catch, I think I was overzealous in removing this.